coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -467,6 +476,20 @@
 
             </div>
           </div>
+          <v-alert
+          close-text="Close Alert"
+          dismissible
+          icon="mdi-information-outline"
+          class="white-background"
+          :id="'director-' + director.id + '-alert'"
+          v-if="complianceMsg && index == messageIndex"
+          v-once
+          >
+            <div class="complianceCard">
```

Comment:
This isn't technically a "card" in Vuetify speak (contrast this with an actual "v-card" element). Please rename it.